Source/XUnitExtensions/Lit/OutputCheckCommand.cs

```diff
@@ -0,0 +1,115 @@
+using System;
```

Comment:
@robin-aws what are all these new check warnings I see on PRs now? Is this something related to this branch?